frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html

```diff
@@ -0,0 +1,169 @@
+                        <li class="bundle-offering-a__subscribe__offer__list__item">
+                            <div class="bundle-offering-a__subscribe__offer__list__item__icon">
+                                @for(icon <- Asset.inlineSvg("bundle-adfree")) {
+                                    @icon
+                                }
+                            </div><div class="bundle-offering-a__subscribe__offer__list__item__content">
+                                <h1>Ad-free</h1>
+                                <p>Faster-loading pages and a clearer reading experience on our website and app</p>
+                            </div>
+                        </li>
+                        <li class="bundle-offering-a__subscribe__offer__list__item">
+                            <div class="bundle-offering-a__subscribe__offer__list__item__icon">
+                                @for(icon <- Asset.inlineSvg("bundle-commenting")) {
+                                    @icon
+                                }
+                            </div><div class="bundle-offering-a__subscribe__offer__list__item__content">
+                                <h1>Commenting</h1>
+                                <p>Discussion threads on articles are restricted to those who become Supporters</p>
+                            </div>
+                        </li>
+                        <li class="bundle-offering-a__subscribe__offer__list__item">
+                            <div class="bundle-offering-a__subscribe__offer__list__item__icon">
+                            @for(icon <- Asset.inlineSvg("bundle-tabletedition")) {
+                                @icon
+                            }
+                            </div><div class="bundle-offering-a__subscribe__offer__list__item__content">
+                                <h1>Tablet edition</h1>
+                                <p>Your daily newspaper optimised for tablet</p>
+                            </div>
+                        </li>
+                        <li class="bundle-offering-a__subscribe__offer__list__item">
+                            <div class="bundle-offering-a__subscribe__offer__list__item__icon">
+                            @for(icon <- Asset.inlineSvg("bundle-newspaper")) {
+                                @icon
+                            }
+                            </div><div class="bundle-offering-a__subscribe__offer__list__item__content">
+                                <h1>Print</h1>
+                                <p>Pick the package you want: Saturday, Sunday, Weekly, Weekend or Everyday.  Save up to 36% on the Guardian and Observer newspapers</p>
+                            </div>
+                        </li>
+                    </ul>
+                    @fragments.bundle.bundleButton("See options","#","", chevron = true)
+                    <div class="bundle-offering-a__subscribe__offer__print-options">
```

Comment:
That `div` has the purpose of wrapping the entire section of print options (`ul` and the `button`). In that way, in the future we will be able to show / hide that entire section.